content/en/blog/_posts/2024-08-19-pod-failure-policy-ga.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: blog
+title: "Kubernetes 1.31: Pod failure policy goes GA"
+date: 2024-08-19
+slug: kubernetes-1-31-pod-failure-policy-stable
+author: >
+  Michał Woźniak (Google)
+  Shannon Kularathna (Google)
+---
+
+This post describes Pod Failure Policy, which graduates to stable in Kubernetes
+1.31, and how to use it in your Jobs.
+
+## About Pod Failure Policy
+
+When you run workloads on Kubernetes, Pods might fail for a variety of reasons.
+Ideally, workloads like Jobs should be able to ignore transient, retriable
+failures and continue running to completion.
+
+To allow for these transient failures, Kubernetes Jobs include the `backoffLimit`
+field, which lets you specify a number of Pod failures that you're willing to tolerate
+during Job execution. However, if you set a large value for the `backoffLimit` field
+and rely solely on this field, you might notice unnecessary increases in operating
+costs as Pods restart excessively until the backoffLimit is met.
+
+This becomes particularly problematic when running large-scale Jobs with
+thousands of long-running Pods across thousands of nodes.
+
+The Pod Failure Policy extends the backoff limit mechanism to help you reduce
```

Comment:
```suggestion
Through pod failure policy, Kubernetes uses an extended backoff limit mechanism; this tries to help you reduce
```
?

Reply:
I'm not convinced this reads better, wdyt @shannonxtreme ?

Reply:
For now I have aligned the casing as "Pod failure policy" since this is the most consistent version with the rest of the blog.

Reply:
Hm. Maybe 

"Through Pod failure policy, you get more control over the Kubernetes backoffLimit mechanism, which helps you to reduce..."? 

Is the issue with the current sentence the word "extends"?

Reply:
I think I like the current version for conciseness, but out of the two proposals, I prefer the latter. wdyt @sftim ?